applications/DEMApplication/python_scripts/DEM_procedures.py

```diff
@@ -159,8 +158,8 @@ def UpdateData(self, domain_volume):
 
         self.voids_volume = domain_volume - self.solid_volume
         self.global_porosity = self.voids_volume / domain_volume
-        if self.do_print_results_option:
-            self.PrintCurrentData()
+
+        self.PrintCurrentData()
```

Comment:
Is this function being called always? Or is it just available for future calls? We don't want to print files always...

Reply:
Thanks! This function only works at the begining and prints once. I think we can leave it here.